### PR TITLE
Adds spoons and forks!

### PR DESCRIPTION
**Pull request by Pontenerd**
## About The Pull Request
Title

## Why It's Good For The Game

Being able to eat with a fork is a fun RP thing

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
add: added spoons and forks
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
Amazing. Speedmerge immediately

**Comment by LodedDiper69** [comment]
Can we see screenshots of those in action? Does this have inhand sprites too?

**Comment by Pontenerd** [comment]
> Can we see screenshots of those in action? Does this have inhand sprites too?

I tried to post a video but GH wasn't having it, 
image
and yes they have inhands

**Comment by WrillWasTaken** [comment]
What happens if I stick it in a light outlet? It'd be funny if it did increased damage compared to other items.

**Comment by Pontenerd** [comment]
> What happens if I stick it in a light outlet? It'd be funny if it did increased damage compared to other items.

it would be funny but it took me 5 hours to code spoons so I don't wanna touch this again

File: code/game/objects/items/kitchen.dm [hunk 1]
```diff
@@ -28,33 +28,50 @@
 	attack_verb = list("attacked", "stabbed", "poked")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30, "stamina" = 0)
+	var/datum/reagent/forkload //used to eat any food (who coded this to just work for omlettes?)
+
+/obj/item/kitchen/fork/suicide_act(mob/living/carbon/user)
```
Comment (EvilDragonfiend) [inline code/game/objects/items/kitchen.dm:33]: duplicating? you have the same proc on this item.

File: code/game/objects/items/kitchen.dm [hunk 2]
```diff
@@ -28,33 +28,50 @@
...
+	throw_speed = 3
+	throw_range = 5
+	materials = list(/datum/material/iron=80)
+	flags_1 = CONDUCT_1
+	attack_verb = list("attacked", "scooped",)
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30, "stamina" = 0)
 	var/datum/reagent/forkload //used to eat omelette
 
 /obj/item/kitchen/fork/suicide_act(mob/living/carbon/user)
```
Comment (EvilDragonfiend) [inline code/game/objects/items/kitchen.dm:69]: this one

**Comment by CydiaLamiales** [comment]
Please tell me this allows us to bypass the limit on food you can normally eat (when you get the 'you cannot shove any more of X down your throat' messages)

**Comment by Bokkiewokkie** [comment]
> Please tell me this allows us to bypass the limit on food you can normally eat (when you get the 'you cannot shove any more of X down your throat' messages)

It does!

File: code/game/objects/items/kitchen.dm (hunk 2 above)
Comment (Bokkiewokkie) [reply code/game/objects/items/kitchen.dm:69]: Probably forgot to change one of these to be the spoon's suicide_act() if I had to guess

**Review by itsmeow — CHANGES_REQUESTED** [review]
Please address reviews (duplicate proc)

**Pull request closed without merging**